<launch>

  <!-- Feature extraction node -->
  <node name="feature_extraction_node" pkg="feature_extraction" type="extraction" output="screen">

    <!-- Spatial thresholds (used for filter point clouds) -->
    <param name="min_x" type="double" value="0.0" />
    <param name="max_x" type="double" value="1.2" />
    <param name="min_y" type="double" value="0.0" />
    <param name="max_y" type="double" value="0.62" />
    <param name="min_z" type="double" value="-0.1" />
    <param name="max_z" type="double" value="0.5" />
    
  </node>

</launch>